lib/modem_info/modem_info.c

```diff
@@ -944,12 +944,33 @@ int modem_info_get_current_band(uint8_t *band)
 	}
 
 	if (*band == BAND_UNAVAILABLE) {
-		return -ENOMSG;
+		return -ENOENT;
 	}
 
 	return 0;
 }
 
+int modem_info_get_snr(int *snr)
+{
+	if (snr == NULL) {
+		return -EINVAL;
+	}
+
+	int ret = nrf_modem_at_scanf("AT%XSNRSQ?", "%%XSNRSQ: %d,%*d,%*d", snr);
```

Comment:
I'm not familiar with the `nrf_modem_at_scanf` api, is this converting multiple `%d` values?

Also big miss on nordic's part to not have the attribute!
https://gcc.gnu.org/onlinedocs/gcc/Common-Function-Attributes.html#index-format-function-attribute

I just learned about the `-Wsuggest-attribute=format` arg to gcc, super neat!